src/main/java/Processor.java

```diff
@@ -0,0 +1,136 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Processor {
+	private List<Task> store = new ArrayList<>();
```

Comment:
```suggestion
	private List<Task> store = new ArrayList<>();
```
Should this be named more specific?